start_signon.sh

```diff
@@ -0,0 +1,34 @@
+#! /bin/bash
+
+set -ex
+
+GEM_ROOT=$(cd $(dirname $0) ; pwd)
+TMP_ROOT=${GEM_ROOT}/tmp
+APP_ROOT=${TMP_ROOT}/signonotron2
+PID_FILE=${APP_ROOT}/server.pid
```

Comment:
In a Rails app I'd normally look in `tmp/pids`. Although this is headless it might be worth following the convention?